setup.py

```diff
@@ -45,6 +45,7 @@ def run(self):
         'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
         'Framework :: Django :: 2.0',
+        'Framework :: Django :: 2.1',
```

Comment:
Unfortunately, this classifier has not been accepted in PyPI's list yet, so this would cause a new PyPI upload to be rejected until that gets fixed.

Reply:
Yep, waiting on in https://github.com/pypa/warehouse/issues/4090.

Reply:
The classifier has been accepted into PyPI's list now.